src/Model/Translatable/TranslatableMethodsTrait.php

```diff
@@ -11,9 +11,6 @@
 
 trait TranslatableMethodsTrait
 {
-    /**
-     * @return Collection|TranslationInterface[]
-     */
     public function getTranslations()
```

Comment:
This should be returned, as it weakens PHPStorm autocomplete abilities

Reply:
We already have the PHPDoc on the [TranslatableInterface](https://github.com/Kocal/DoctrineBehaviors/blob/generics/translatable-translation/src/Contract/Entity/TranslatableInterface.php#L14-L18), it does not break PHPStorm autocomplete:
![Capture d’écran de 2020-02-11 21-34-12](https://user-images.githubusercontent.com/2103975/74276592-4b875100-4d16-11ea-8b8f-2f5747d9a377.png)

Reply:
How about human readability?